apps/happychat/src/happychat.jsx

```diff
@@ -1,194 +0,0 @@
-import { useEffect, useState } from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import Composer from 'calypso/components/happychat/composer';
-import HappychatConnection from 'calypso/components/happychat/connection-connected';
-import Notices from 'calypso/components/happychat/notices';
-import Timeline from 'calypso/components/happychat/timeline';
-import { isOutsideCalypso } from 'calypso/lib/url';
-import { getCurrentUser } from 'calypso/state/current-user/selectors';
-import {
-	sendEvent,
-	sendMessage,
-	sendNotTyping,
-	sendTyping,
-	sendUserInfo,
-	setChatCustomFields,
-	setChatTag,
-} from 'calypso/state/happychat/connection/actions';
-import {
-	HAPPYCHAT_CONNECTION_STATUS_CONNECTING,
```

Comment:
We can totally remove all those `HAPPYCHAT_` action types too:

https://github.com/Automattic/wp-calypso/blob/19429264405a78f97a4b5188a84f59c0b3f5e193/client/state/action-types.ts#L256-L286

Reply:
Nice find! No idea why my searches did not pick that up 🤦 Removed in https://github.com/Automattic/wp-calypso/pull/78433/commits/a4d654b99c671b7bd38f8524b28907ffc57e441f.